cypress/integration/instructors.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('instructors page', () => {
+  before(() => {
+    cy.loginAndVisit();
+  });
+
+  it('navigates to the Assign Instructors page', () => {
+    cy.contains('Instructors').click();
+    cy.contains('Assign Instructors').click();
+    cy.location().should(loc => {
+      expect(loc.pathname).to.contains('assignments');
+    });
+    cy.contains('Assignments');
+  });
+
+  it('assigns instructor to ECS10', () => {
+    cy
+      .contains('ECS 010')
+      .parents('div.course-list-row')
+      .as('row')
+      .within($row => {
+        cy.get('.dropdown').click();
+
+        cy.server();
+        cy.route('POST', '**/assignmentView/**').as('postInstructor');
+
+        cy.contains('Jarold Wong').click();
```

Comment:
Here's another example of a "specific string" (magic string) to turn into a variable at the top of this case (so, variable declaration on line 16)